accessibility/analyze.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+import argparse
+import sys
+import geopandas as gpd
+import pandas as pd
+from shapely.geometry import Point
+from rasterstats import zonal_stats
+from pyproj.aoi import AreaOfInterest
+from pyproj.database import query_utm_crs_info
+from pathlib import Path
+
+parser = argparse.ArgumentParser(
+    description="Run zonal stats on area defined by set of isochrone polygons")
+parser.add_argument("input", help="geojson file containing polygon collection")
+parser.add_argument("--pop_tiff", required=True,
+                    help="path to population GeoTIFF")
+parser.add_argument("--buffer_distance", type=int, default=1,
+                    help="Buffer geometries to incorporate walking time to roads and fix invalid isochrones")
+parser.add_argument("--limit", default=None, type=int,
+                    help="If provided, load only some features for testing")
+parser.add_argument("--no_outputs", default=False,
+                    action="store_true", help="Don't save results to a file")
+parser.add_argument("--output", required=not '--no_outputs' in sys.argv,
```

Comment:
This is confusing -- why are there both options for `-output` and `--no_outputs?`. 

Also just realized we are using underscores here we should probably rewrite these as hyphens for consistency with [convention](https://developer.akamai.com/cli/docs/style-guide)